### PR TITLE
UX cosmetics for users, groups, service accounts

### PR DESCRIPTION
**Pull request by prakashsvmx**
Impacted Screens ( Add/Edit/Delete)

Users
Service Accounts
Groups

UX
https://xd.adobe.com/view/f39e14b7-5d46-4dc6-a6da-d4e37f2f6836-a6cd/

### CODE REVIEW

**Review by dvaldivia — APPROVED** [review]
Tested, we need to fix icons, but it's not related to this PR. LGTM

File: portal-ui/src/screens/Console/Groups/GroupsDetails.tsx [hunk 1]
```diff
@@ -102,11 +131,18 @@ const GroupsDetails = ({ classes }: IGroupDetailsProps) => {
   const [policyOpen, setPolicyOpen] = useState<boolean>(false);
   const [usersOpen, setUsersOpen] = useState<boolean>(false);
   const [deleteOpen, setDeleteOpen] = useState<boolean>(false);
+  const [memberFilter, setMemberFilter] = useState<string>("");
+
+  //const [policyFilter, setPolicyFilter] = useState<string>("");
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Groups/GroupsDetails.tsx:136]: Please remove this
```suggestion

```

File: portal-ui/src/screens/Console/Groups/AddGroup.tsx [hunk 2]
```diff
@@ -193,41 +199,42 @@ const AddGroup = ({
...
+                />
+              </Grid>
             ) : (
               <React.Fragment>
                 <PredefinedList label={"Group Name"} content={selectedGroup} />
               </React.Fragment>
             )}
             <Grid item xs={12}>
               <UsersSelectors
+                classes={classes}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Groups/AddGroup.tsx:220]: This classes may be called inside UsersSelectors component and then we can remove this prop

File: portal-ui/src/screens/Console/Groups/UsersSelectors.tsx [hunk 3]
```diff
@@ -54,60 +53,30 @@ const styles = (theme: Theme) =>
       display: "flex",
       overflow: "auto",
       flexDirection: "column",
-      paddingTop: 15,
+      // paddingTop: 15,
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Groups/UsersSelectors.tsx:56]: We can remove this
```suggestion
      
```

File: portal-ui/src/screens/Console/Policies/SetPolicy.tsx [hunk 4]
```diff
@@ -137,32 +150,34 @@ const SetPolicy = ({
...
       <Grid item xs={12}>
         <PredefinedList
+          classes={classes}
           label={`Selected ${selectedGroup !== null ? "Group" : "User"}`}
           content={selectedGroup !== null ? selectedGroup : userName}
         />
       </Grid>
       <Grid item xs={12}>
         <PredefinedList
+          classes={classes}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Policies/SetPolicy.tsx:160]: Same case, this classes can be placed inside Predefined List (please use HOC) and then we can remove this prop

File: portal-ui/src/screens/Console/Users/ChangeUserGroups.tsx [hunk 5]
```diff
@@ -163,26 +167,25 @@ const ChangeUserGroups = ({
...
-              <Grid item xs={12}>
-                <GroupsSelectors
-                  selectedGroups={selectedGroups}
-                  setSelectedGroups={(elements: string[]) => {
-                    setSelectedGroups(elements);
-                  }}
-                />
-              </Grid>
+              <GroupsSelectors
+                classes={classes}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ChangeUserGroups.tsx:171]: Same case

File: portal-ui/src/screens/Console/Users/ListUsers.tsx [hunk 6]
```diff
@@ -229,23 +230,10 @@ const ListUsers = ({ classes, setErrorSnackMessage, history }: IUsersProps) => {
...
-              ),
-            }}
-            onChange={(e) => {
-              setFilter(e.target.value);
-            }}
-            variant="standard"
+          <SearchBox
+            placeholder={"Search Groups"}
+            onChange={setFilter}
+            classes={classes}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/ListUsers.tsx:236]: same case

File: portal-ui/src/screens/Console/Users/UserDetails.tsx [hunk 7]
```diff
@@ -364,14 +362,16 @@ const UserDetails = ({ classes, match }: IUserDetailsProps) => {
...
-                    // itemActions={userTableActions}
-                    columns={[{ label: "Name", elementKey: "group" }]}
-                    isLoading={loading}
-                    records={currentGroups}
-                    entityName="Groups"
-                    idField="group"
-                  />
+                  <div className={classes.tableBlock}>
+                    <TableWrapper
+                      // itemActions={userTableActions}
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Users/UserDetails.tsx:367]: Please remove
```suggestion

```

**Review by bexsoft — CHANGES_REQUESTED** [review]

**Review by bexsoft — APPROVED** [review]
Will Make the requested changes in a new PR

**Pull request merged**